empty commit

IkmosaO opened the pull request:
### goal of the PR
The goal of this PR is to set a basic route with a skeleton.

### What changes were introduced, and the expected behavior
I have created the routes for the customer, and I have also created the skeletons for the controllers.
The expected behavior is to be able to post through postman to the customer routes, and not see any errors.

IkmosaO committed:
empty commit

IkmosaO committed:
updated index.js with new routes

IkmosaO commented:
Please check out my index.js, added a few things to file. What do you guys think?
@grimmi5894 @catb0591 @JesusOrozco978 @IgnacioOAF

IkmosaO committed:
updated small errors to index.js and controller

IkmosaO committed:
added comments to each route describing we think they should do

IgnacioOAF commented on controller/index.js:
```diff
@@ -0,0 +1,44 @@
+    getContactById,
+    uploadContacts,
+    uploadCSVFile
+}
```
Nit: here you need to add an empty line.

IgnacioOAF reviewed (approved):
Great job here

catb0591 reviewed (commented):
Going to look at this!

IgnacioOAF reviewed (changes requested):
change this

catb0591 reviewed (approved):
This looks great @IkmosaO 
Let me know when this is merged so I can pull this up and start tackling some of the controllers :)

catb0591 commented:
@IgnacioOAF - are we okay to merge this when @IkmosaO adds the space at the end?

Ignacio01 commented:
Yes, this should be safe to merge.

IkmosaO committed:
changed main index.js filename (lowcased the first letter), fixed spelling errors

IkmosaO commented:
Alright i added that space, fixed a spelling error in the controllers/index.js  on line 27, and did some formatting.

I'll be merging this branch to main now.
